{"name":"Update: Failure","fullName":"TestPublicationPgRepoSuiteRunner/PublicationPgRepoSuit/TestPublicationPgRepo_UpdateFailure","status":"passed","statusDetails":{"message":"","trace":""},"start":1728207674847,"stop":1728207674847,"uuid":"2113dc76-83c7-11ef-8b51-be2bab8c44a2","historyId":"c748530daa8d346074db03fbbb74e415","testCaseId":"d9c71ea48805fe7d43cd47d2585a3e1e","labels":[{"name":"language","value":"go1.23.0"},{"name":"framework","value":"Allure-Go@v0.6.0"},{"name":"host","value":"MBP-Pavel-2.netis.cc"},{"name":"thread","value":"TestPublicationPgRepoSuiteRunner/PublicationPgRepoSuit/TestPublicationPgRepo_UpdateFailure"},{"name":"suite","value":"PublicationPgRepoSuit"},{"name":"package","value":"cookdroogers/internal/repo/pg"},{"name":"tag","value":"PublicationPgRepo"}],"steps":[{"name":"Failure","status":"passed","start":1728207674847,"stop":1728207674847,"steps":[{"name":"ASSERT: Error Is","status":"passed","start":1728207674847,"stop":1728207674847,"parameters":[{"name":"Error","value":"sql: connection is already closed: error in postgres repo"},{"name":"Target","value":"error in postgres repo"}]}]}]}